<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorDialog</class>
 <widget class="QDialog" name="ColorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Palette Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>400</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="paletteSelectionGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>501</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>Palette Selection</string>
   </property>
   <widget class="PaletteListView" name="listView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>481</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="newPaletteButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>New</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deletePaletteButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>141</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Selection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveToFileButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save (*.pal)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadFromFileButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>170</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Load (*.pal)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="palettePreviewGroupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>501</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Palette Preview</string>
   </property>
   <widget class="ColorSelectorWidget" name="colorWidget0" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget1" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget2" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget3" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget4" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget5" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget6" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget7" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget8" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget12" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget14" native="true">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget13" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget10" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget9" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget15" native="true">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget11" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget11_2" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget10_2" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget13_2" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget0_2" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget8_2" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget12_2" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget14_2" native="true">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget4_2" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget6_2" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget15_2" native="true">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget3_2" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget7_2" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget9_2" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget1_2" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget2_2" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget5_2" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget11_3" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget9_3" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget5_3" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget1_3" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget3_3" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget0_3" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget13_3" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget6_3" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget10_3" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget4_3" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget12_3" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget7_3" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget15_3" native="true">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget14_3" native="true">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget8_3" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget2_3" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget11_4" native="true">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget6_4" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget3_4" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget2_4" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget9_4" native="true">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget15_4" native="true">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget14_4" native="true">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget10_4" native="true">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget5_4" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget12_4" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget13_4" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget4_4" native="true">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget8_4" native="true">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget7_4" native="true">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget1_4" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="ColorSelectorWidget" name="colorWidget0_4" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaletteListView</class>
   <extends>QListView</extends>
   <header>src/ColorDialog/PaletteListView.h</header>
  </customwidget>
  <customwidget>
   <class>ColorSelectorWidget</class>
   <extends>QWidget</extends>
   <header>src/ColorDialog/ColorSelectorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ColorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ColorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
